frontend/src/meeting-detail/RegistrationForm.jsx

```diff
@@ -85,6 +87,7 @@ export default function RegistrationForm({ isDisabled = false }) {
               value={emailAddress}
               onChange={(e) => setEmailAddress(e.target.value)}
             />
+            {isShowValidated && emailAddress === '' ? <div className="invalid">Email can not be empty.</div>:''}
```

Comment:
Hopefully you could write a full email address validator instead of just checking if the email field is empty. Users may enter a wrong email address, such as `name@gmail,com`.
You can try googling or asking ChatGPT how to write an email validator with React.js.